lib/jekyll-compose/file_editor.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+#
+# This class is aimed to open created file in selected editor.
+# To use this feature specify at Jekyll config:
+#
+# ```
+#  jekyll_compose:
+#    auto_open: true
+# ```
+#
+# And make sure, that you have JEKYLL_EDITOR or EDITOR env variables set up.
+# This will allow to open file in vim editor automatically.
```

Comment:
✂️ this isn't limited to Vim

Reply:
There are other editors? 🤔

Reply:
Haha, didn't wanted to start **this**  discussion 😆